pilot/pkg/features/pilot.go

```diff
@@ -288,6 +288,13 @@ var (
 		"If this is set to true, support for Kubernetes service-apis (github.com/kubernetes-sigs/service-apis) will "+
 			" be enabled. This feature is currently experimental, and is off by default.").Get()
 
+	EnableVirtualServiceDelegate = env.RegisterBoolVar(
```

Comment:
nit: '.Get()` here it self

Reply:
This is to support dynamically enable/disable the feature without a restart.

Reply:
But do we need that?  Would n't some proxies have delegate enabled and some have delegate disabled if we dynamically enable/disable it?

Reply:
Missed it, i donot mind fixing it now. It is rare to use like this.